src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Announcing Meshery v0.5.0"
+subtitle: ""
+date: 2021-01-25 08:00:00 +0000
+author: Aisuko Li
+thumbnail: ./meshery.png
+category: Announcements
+tags:
+ - Meshery
+ - Projects
+ - News
+published: true
+redirect_from:
+ - /blog/meshery/announcing-meshery-v0.5.0/
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import operator from "./meshery-operator-dark.svg";
+
+<BlogWrapper>
+
+<p style="color:#3c494f;text-align:center;">Thanks to our open source contributors</p>
+
+We're pleased to announce the realease of Meshery 0.5.0, our first release of 2021! This realease consists of number of significant architectural enhancements and 8 new capabilities are entering alpha.
+
+The 0.5.0 release cycle will good begin for 2021 release cycle, this is one of the most feature dense releases in a while:the Meshery innovation
+cycle is still trending upward. This release has many new capabilities,showing that there is still much to explore in the open service mesh ecosystem.
+
+
+## Major themes
+
+The two new adapters with the latest version of Meshery released
+
+* Meshery Adapter for  <a href="https://github.com/layer5io/meshery-traefik-mesh">Traefik Mesh (beta)</a>
+* Meshery Adapter for <a href="https://github.com/layer5io/meshery-nginx-sm">NGINX SM Mesh (beta)</a>
+
+* Dynamic injection of Remote Provider extensions
+  * Need more information
```

Comment:
Need more inforamtion

Reply:
Here we are - https://docs.meshery.io/extensibility/providers